### PR TITLE
Added the Main Page and Other Features

### PR DESCRIPTION
**Pull request by marinasemen0va**
Added: 
- custom theme elements
- login/sign up buttons
- subjects dropdown
- main page
- incomplete subject pages (maybe temporary)

### CODE REVIEW

File: content/chemistry/chemistry.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: default
```
Comment (0ffz) [inline content/chemistry/chemistry.md:2]: Consider making a `subject` layout

File: index.md [hunk 2]
```diff
@@ -1,10 +1,7 @@
...
+layout: home
+title: Home
 permalink: /
 ---
 
-# Under construction!
-
-$$ \frac{1}{2} $$
\ No newline at end of file
+{% include main.html %}
```
Comment (0ffz) [inline index.md:7]: Rename file to index.html and just move main.html in here

**Review by 0ffz — CHANGES_REQUESTED** [review]

File: content/chemistry/chemistry.md (hunk 1 above)
Comment (marinasemen0va) [reply content/chemistry/chemistry.md:2]: Yes, we will consider this.

File: index.md (hunk 2 above)
Comment (marinasemen0va) [reply index.md]: Done.

**Pull request merged**